Gitagram/View/AccountView/View/AccountView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  AccountQRView.swift
+//  Gitagram
+//
+//  Created by 伊藤璃乃 on 2024/05/03.
+//
+
+import SwiftUI
+import CoreImage.CIFilterBuiltins
+
+struct AccountFrameView: View {
+    private static let cardCornerRadius: CGFloat = 20
+    private static let cardSize: CGSize = .init(width: 367, height: 512)
+    //カードの大きさ
+    @State private var github = "https://github.com/"
```

Comment:
あとこの辺り、Developer型でまとめちゃって欲しいかも！
```suggestion
@State private var developer = Developer(githubId: "unknown")


~~~~~
// 以下のようにするとGitHubのURL取れる
developer.githubURL 
```